### PR TITLE
📝 Document new write-discussions oauth scope

### PR DESCRIPTION
**Pull request by coyotte508**

### CODE REVIEW

File: docs/hub/oauth.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The currently supported scopes are:
 - `write-repos`: Get write/read access to the user's personal repos.
 - `manage-repos`: Get full access to the user's personal repos. Also grants repo creation and deletion.
 - `inference-api`: Get access to the [Inference API](https://huggingface.co/docs/api-inference/index), you will be able to make inference requests on behalf of the user.
+- `write-discussions`: Open discussions and pull requests on behalf of the user, and interact with discussions.
```
Comment (osanseviero) [inline docs/hub/oauth.md:38]:
```suggestion
- `write-discussions`: Open discussions and Pull Requests on behalf of the user as well as interact with discussions.
```

File: docs/hub/spaces-oauth.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ Those scopes are optional and can be added by setting `hf_oauth_scopes` in your
 - `write-repos`: Get write/read access to the user's personal repos.
 - `manage-repos`: Get full access to the user's personal repos. Also grants repo creation and deletion.
 - `inference-api`: Get access to the [Inference API](https://huggingface.co/docs/api-inference/index), you will be able to make inference requests on behalf of the user.
+- `write-discussions`: Open discussions and pull requests on behalf of the user, and interact with discussions.
```
Comment (osanseviero) [inline docs/hub/spaces-oauth.md:79]:
```suggestion
- `write-discussions`: Open discussions and Pull Requests on behalf of the user as well as interact with discussions.
```

File: docs/hub/oauth.md (hunk 1 above)
Comment (osanseviero) [inline docs/hub/oauth.md:38]: What does interact mean? Reactions? Replying to?

**Review by osanseviero — APPROVED** [review]

File: docs/hub/oauth.md (hunk 1 above)
Comment (coyotte508) [reply docs/hub/oauth.md:38]: yes, reactions, comments, editing comments, closing discussions, changing the title, ...

Comment (julien-c) [reply docs/hub/oauth.md]: add this between parenthesis?

**Review by julien-c — APPROVED** [review]

**Pull request merged**